src/lib/features/feature-toggle/feature-toggle-service.ts

```diff
@@ -390,19 +390,25 @@ class FeatureToggleService {
         }
     }
 
-    validateConstraintsLimit(updatedConstrains: IConstraint[]) {
+    validateConstraintsLimit(constraints: {
+        updated: IConstraint[];
+        existing: IConstraint[];
```

Comment:
Because these parameters have the same type, I want to force people to name them to avoid confusion. That's also the reason why I've marked both as required.